addons/website_slides/controllers/main.py

```diff
@@ -45,6 +45,7 @@ def _slide_render_context_base(self):
             # current user info
             'user': request.env.user,
             'is_public_user': request.website.is_public_user(),
+            'is_internal_user': request.website.is_user(),
```

Comment:
Seems very strange as this does not really test any "internal" behavior from what I can see in the code.

How about a regular `request.env.user._is_internal()` ?